pages/linux/vnstati.md

```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> PNG image output support for vnStat.
+> More information: <https://manned.org/vnstati>.
+
+- Output summary of the last 2 days/months and all time:
```

Comment:
```suggestion
- Output a summary of the last 2 months, days, and all time:
```

How is it chosen for whether it is days or months
if it is 1 or the other then `/` is wrong

Reply:
It shows the days and the months, so both. [Example output](https://i.imgur.com/exDJ0WI.png).